pkg/common/config/config_test.go

```diff
@@ -33,6 +33,26 @@ datacenters = us-west
 ca-file = /some/path/to/a/ca.pem
 `
 
+const twoVCsUsingSecretConfig = `
+[Global]
+port = 443
+insecure-flag = true
+
+[VirtualCenter "us-west"]
+server = "10.0.0.1"
+datacenters = "vic0dc"
+secret-name = "us-west-secret"
+secret-namespace = "kube-system"
+# port, insecure-flag will be used from Global section.
+
+[VirtualCenter "us-east"]
```

Comment:
Can we have a 3rd entry here that still uses the server IP as the connection label?
```
[VirtualCenter "10.0.0.3"]
datacenters = "vic0dc"
secret-name = "us-west-secret"
secret-namespace = "kube-system"
# port, insecure-flag will be used from Global section.
```